openlxp_authentication/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.db.models.base import Model
+from django.conf import settings
+from social_core.backends.saml import SAMLAuth, SAMLIdentityProvider
+from social_django.strategy import DjangoStrategy
+
+
+class SAMLConf(Model):
+    name = models.CharField(unique=True, max_length=50)
+    entity_id = models.CharField(max_length=300)
+    url = models.URLField()
+    cert = models.TextField()
+
+    def attributes():
+        return dict.fromkeys(["attr_" + attr for attr in settings.USER_ATTRIBUTES], '')
+    attribute_mapping = models.JSONField(default=attributes)
+
+
+class SAMLDBAuth(SAMLAuth):
```

Comment:
I understand that we are subclassing the SAMLAuth object here, but It's not clear to me why `DB` is relevant -- is there some other term we can use to do a better job of conveying the point of this subclass.  Are we trying to say that this model is database-oriented, whereas the parent class is not?

Reply:
Correct, the parent class uses configurations stored in the Django settings file.  This subclass uses
the SAMLConfigurations stored in the database

Reply:
Cool, the name makes sense, then.  Let's include this context in a docstring for the class.